staging_docs/admin/guides/build-image.md

```diff
@@ -41,12 +41,18 @@ CMD ["cat", "/inside-image.txt"]' >> Containerfile
 ## Build an OCI image
 
 ```bash
-TASK_HREF=$(http --form POST :$REPO_HREF'build_image/' containerfile@./Containerfile \
-artifacts="{\"$ARTIFACT_HREF\": \"foo/bar/example.txt\"}"  | jq -r '.task')
+ARTIFACT_SHA256=$(http :$ARTIFACT_HREF | jq -r '.sha256')
```

Comment:
We no longer care about artifacts, do we?

Reply:
Hum ... we don't, I kept this artifact just as a sample/doc to populate the repository:
https://github.com/pulp/pulp_container/pull/1687/files/010d8fb88c2361abbe8f4cefc5a6b62ba42308ee#diff-8d94fc0843f51b6f430b79c609efee42205be12f7fee3369c1dd7824eeabe779R47-R48

I'll update it to create the file content "directly".

Reply:
Please do. We should forget about using the artifacts endpoint.